third_party/move/move-compiler/src/cfgir/cfg.rs

```diff
@@ -192,6 +192,72 @@ impl<'a> BlockCFG<'a> {
             println!();
         }
     }
+
+    /// Adds a block on any critical edge: an edge from a node with multiple successors to
+    /// a node with multiple predecessors.  Returns true if any changes occur; in this
+    /// case, caller must call recompute().
```

Comment:
Perhaps clarify recompute means something with the overall graph. I first thought you mean recompute for splitting edges again, which seems to be not the case.

Reply:
clarified.